CODEOWNERS

```
@@ -87,6 +87,7 @@ esphome/components/bp1658cj/* @Cossid
 esphome/components/bp5758d/* @Cossid
 esphome/components/button/* @esphome/core
 esphome/components/bytebuffer/* @clydebarrow
+esphome/components/camera/* @DT-art1 @bdraco
```

```suggestion
esphome/components/camera/* @DT-art1
```

No need to add me to this. I already watch all the issue queue anyways